src/dns_public.tf

```diff
@@ -38,6 +38,27 @@ resource "azurerm_dns_ns_record" "uat_pagopa_it_ns" {
   tags = var.tags
 }
 
+resource "azurerm_dns_caa_record" "api_platform_pagopa_it" {
```

Comment:
also add caa record for https://github.com/pagopa/pagopa-infra/blob/main/src/dns_checkout_public.tf

Reply:
done 👍